internal/querycoord/cluster.go

```diff
@@ -374,9 +374,22 @@ func (c *queryNodeCluster) addQueryChannel(ctx context.Context, nodeID int64, in
 	c.RUnlock()
 
 	if targetNode != nil {
-		err := targetNode.addQueryChannel(ctx, in)
+		emptyChangeInfo := &querypb.SealedSegmentsChangeInfo{
+			Base: &commonpb.MsgBase{
+				MsgType: commonpb.MsgType_SealedSegmentsChangeInfo,
+			},
+		}
+		msgPosition, err := c.clusterMeta.sendSealedSegmentChangeInfos(in.CollectionID, in.QueryChannel, emptyChangeInfo)
```

Comment:
here we send only empty Change Info, how did the queryNode get full segment list?

Reply:
query node get full segment list only when receiver a watchQueryChannel grpc request